src/julia.h

```diff
@@ -12,6 +12,7 @@ extern "C" {
 #include "libsupport.h"
 #include <stdint.h>
 #include <string.h>
+#include <inttypes.h>
```

Comment:
Good point. Actually, they are only used in files that does not include llvm (`builtins.c`).

Hopefully this version works...